plugins/xpad/xpad.cpp

```diff
@@ -23,6 +23,10 @@
 #include "xpad.h"
 #include <VersionHelpers.h>
 
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x0602 // Required for XInputEnable definition
```

Comment:
The DX SDK build people that aren't using W8+ are compiling against?
In this light I found that "odd".